opensearch/sycamore-opensearch.sh

```diff
@@ -423,9 +386,136 @@ deploy_model() {
     if [[ -z "${model_id}" ]]; then
         wait_task "${task_id}" "$name model to become ready"
         model_id="${GET_TASK_MODEL_ID}"
-        echo "Fetched $name model id $model_id for task_id $task_id"
+        echo "Fetched $name model id $model_id for task_id $task_id" >&2
```

Comment:
I'll recommend changing all of these to 
status "Fetched $name model id $model_id for task_id $task_id"
and writing a 
```
status() {
  echo "$@" >&2
}
```
function.
Requiring people to remember to put >&2 at the end of each line is very error-prone.
```
grep echo sycamore-opensearch.sh | grep -v '>>' | grep -v '>'
```
will find all the ones that were missed.